.claude/agents/flutter-performance-optimizer.md

```
@@ -0,0 +1,680 @@
+
+## Output Standards
+
+Provide:
+1. **Specific code changes** (before/after)
+2. **Expected performance improvement** (quantified)
+3. **Verification steps** to measure improvement
+4. **Priority** (High/Medium/Low impact)
+5. **Risk assessment** (breaking changes)
+
+Example output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)